galsim/__init__.py

```diff
@@ -168,3 +169,6 @@
 from . import dcr
 from . import meta_data
 from . import cdmodel
+
+# This now depends on chromatic stuff.
+
```

Comment:
This is a remnant.  My fault.  I moved the catalog import back where it had come from, since I fixed the dependence on chromatic (which is now in scene), but I forgot to remove this comment.